### DISCUSSION TITLE
Dependency on R >= 4.0

### DISCUSSION THREAD

**Discussion by shikokuchuo** [wlandau/crew#140] [Q&A] [answered]
Just curious as to the reason such a recent version is required. I'm also not aware of many features introduced by that version!

`nanonext`/`mirai` only require R >= 3.5 to ensure R serialization version 3 is present - so that all clients are interoperable. There is also a use of `...length()` in `mirai` although this could trivially be avoided if necessary.

**Comment by wlandau** [comment, marked as answer]
I don't remember. I may have wanted to use `tools::R_user_dir()` at some point, but that must have been a long time ago. Regardless, I think R 4.0.0 is old enough for this restriction to be harmless. Not sure what kind of Chesterton fence I would hit by decreasing the minimum version.

**Reply by shikokuchuo** [reply]
OK. Just wanted to point out that `mirai`/`nanonext` don't have such requirements and (at least theoretically) there could be `targets` users that are prevented from using `crew`.

**Reply by wlandau** [reply]
You bring up a good point. However, requiring 4.0.0 seems unlikely to cause problems in practice, and I don't have time to retest on 3.x.x. If 4.0.0 is too strict at the moment, it seems like a problem that will solve itself over time.
